frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/types.ts

```diff
@@ -45,6 +88,8 @@ export type PipelineMixedNodeModel = PipelineNodeModel<PipelineRunAfterNodeModel
 export type PipelineTaskNodeModel = PipelineNodeModel<TaskNodeModelData>;
 export type PipelineBuilderTaskNodeModel = PipelineNodeModel<BuilderNodeModelData>;
 export type PipelineTaskListNodeModel = PipelineNodeModel<TaskListNodeModelData>;
+export type PipelineFinallyNodeModel = PipelineNodeModel<FinallyNodeModel>;
+export type PipelineFinallyBuilderNodeModel = PipelineNodeModel<BuilderFinallyNodeModel>;
```

Comment:
Do a quick search for all `FinallyBuilder` and `BuilderFinally` usages, and align under one banner please 😄